test/sqllogictest/query_graph.slt

```diff
@@ -0,0 +1,42 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+statement ok
+create table t(f1 int, f2 int);
```

Comment:
Change this to have a primary key so that we can see the unique key print out?

Reply:
My idea with this test was to just test that the command works using the simplest graph using the least amount of features so that it doesn't have to be updated too often.

Reply:
Ok. Add a comment to the top of this file saying that this file exists to test that `EXPLAIN QUERY GRAPH` works and that testing for specific outputs can be found in src/sql/tests/query_model.

Reply:
good idea.